pkg/resources/samples/trace-dns.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   node: minikube
   gadget: dns
   filter:
-    namespace: kube-system
+    namespace: gadget
```

Comment:
This should not be modified because the dns pod what we want to filter runs in kube-system. (As already mentioned by Alban).